README.md

```diff
@@ -786,8 +786,8 @@ task jes_task {
     docker: "ubuntu:latest"
     memory: "4G"
     cpu: "3"
-    defaultZones: "US_Metro MX_Metro"
-    defaultDisks: "Disk1 3 SSD, Disk2 500 HDD"
+    zones: "US_Metro MX_Metro"
+    disks: "Disk1 3 SSD, Disk2 500 HDD"
```

Comment:
I hate to create new work, though do you think this PR should be followed by a PR on the dsde-pipelines repo to use these new values?